mmv1/products/siteverification/WebResource.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'WebResource'
+description: |
+  A web resource is a website or domain with verified ownership. Once your
+  ownership is verified you will be able to manage your website in the
+  [Google Search Console](https://www.google.com/webmasters/tools/).
+
+  ~> **Note:** The verification data (DNS `TXT` record, HTML file, `meta` tag, etc.)
+  must already exist before the web resource is created, and must be deleted before
+  the web resource is destroyed. The Google Site Verification API checks that the
+  verification data exists at creation time and does not exist at destruction time
+  and will fail if the required condition is not met.
+base_url: webResource
+self_link: webResource/{{id}}
+create_url: webResource?verificationMethod={{verification_method}}
+delete_url: webResource/{{id}}
+id_format: '{{id}}'
+immutable: true
+identity:
+  - id
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    'Getting Started': 'https://developers.google.com/site-verification/v1/getting_started'
+  api: 'https://developers.google.com/site-verification/v1'
+timeouts: !ruby/object:Api::Timeouts
+  insert_minutes: 60
+error_retry_predicates: ['transport_tpg.IsSiteValidationRetryableError']
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    skip_test: true
+    name: "site_verification_domain_record"
+    primary_resource_id: "example"
+    vars:
+      managed_zone: "example.com"
+      domain: "www.example.com"
+      dns_name: "www.example.com."
+supports_indirect_user_project_override: true
+parameters:
+  - !ruby/object:Api::Type::Enum
+    name: 'verification_method'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The verification method for the Site Verification system to use to verify
+      this site or domain.
+    values:
+      - :ANALYTICS
+      - :DNS_CNAME
+      - :DNS_TXT
+      - :FILE
+      - :META
+      - :TAG_MANAGER
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'id'
+    output: true
+    immutable: true
```

Comment:
immutable is not applicable for an output field

Reply:
Done